environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -224,6 +224,13 @@ FlowIdentifier postStartByCrn(@ValidCrn(resource = CrnResourceDescriptor.ENVIRON
             nickname = "stopEnvironmentByCrnV1")
     FlowIdentifier postStopByCrn(@ValidCrn(resource = CrnResourceDescriptor.ENVIRONMENT) @PathParam("crn") String crn);
 
+    @POST
+    @Path("/name/{name}/rotate_salt_password")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = EnvironmentOpDescription.ROTATE_SALT_PASSWORD_BY_NAME, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
+            nickname = "rotateSaltPasswordEnvironmentByNameV1")
+    void rotateSaltPasswordByName(@PathParam("name") String name);
```

Comment:
is there a reason to use name as parameter instead of crn? with crn we can save lot of db queries usually

Reply:
I was thinking that it will be called by name from cdp cli, but I will implement a name to crn conversion there if that is more efficient

Reply:
if it is needed for cdp cli, we can keep the endpoint with name, or you can decide which solution is better. I am just preferring crn, i would prefer it in cdp cli to be honest, but i know maybe customers like name more